dashboard/app/models/applab.rb

```diff
@@ -107,193 +109,6 @@ def update_json_fields
-        "getY": null,
-        "getDirection": null,
-        "penUp": null,
-        "penDown": null,
-        "penWidth": null,
-        "penColor": null,
-        "penRGB": null,
-        "show": null,
-        "hide": null,
-        "speed": null,
-
-        // Control
-        "forLoop_i_0_4": null,
-        "ifBlock": null,
-        "ifElseBlock": null,
-        "whileBlock": null,
-        "setTimeout": null,
-        "clearTimeout": null,
-        "setInterval": null,
-        "clearInterval": null,
-        "getTime": null,
-
-        // Math
-        "addOperator": null,
-        "subtractOperator": null,
-        "multiplyOperator": null,
-        "divideOperator": null,
-        "equalityOperator": null,
-        "inequalityOperator": null,
-        "greaterThanOperator": null,
-        "greaterThanOrEqualOperator": null,
-        "lessThanOperator": null,
-        "lessThanOrEqualOperator": null,
-        "andOperator": null,
-        "orOperator": null,
-        "notOperator": null,
-        "randomNumber_min_max": null,
-        "mathRound": null,
-        "mathAbs": null,
-        "mathMax": null,
-        "mathMin": null,
-        "mathRandom": null,
-
-        // Variables
-        "declareAssign_x": null,
-        "declareNoAssign_x": null,
-        "assign_x": null,
-        "declareAssign_x_prompt": null,
-        "declareAssign_x_promptNum": null,
-        "console.log": null,
-        "declareAssign_str_hello_world": null,
-        "substring": null,
-        "indexOf": null,
-        "includes": null,
-        "length": null,
-        "toUpperCase": null,
-        "toLowerCase": null,
-        "declareAssign_list_abd": null,
-        "listLength": null,
-        "insertItem": null,
-        "appendItem": null,
-        "removeItem": null,
-
-        // Functions
-        "functionParams_none": null,
-        "functionParams_n": null,
-        "callMyFunction": null,
-        "callMyFunction_n": null,
-        "return": null,
-        "comment": null,
-
-        // Maker
-        "pinMode": null,
-        "digitalWrite": null,
-        "digitalRead": null,
-        "analogWrite": null,
-        "analogRead": null,
-        "timedLoop": null,
-
-        // Circuit Playground
-        "on": null,
-        "off": null,
-        "toggle": null,
-        "blink": null,
-        "stop": null,
-        "color": null,
-        "intensity": null,
-        "led.on": null,
-        "led.off": null,
-        "buzzer.frequency": null,
-        "buzzer.note": null,
-        "buzzer.off": null,
-        "buzzer.play": null,
-        "accelerometer.getOrientation": null,
-        "accelerometer.start": null,
-        "accelerometer.getAcceleration": null,
-        "isPressed": null,
-        "holdtime": null,
-        "value": null,
-        "setScale": null,
-        "start": null,
-        "getAveragedValue": null,
-        "toggleSwitch.isOpen": null,
-        "onBoardEvent": null
-      }
-    JSON
+    SharedConstants::APPLAB_BLOCKS
```

Comment:
Did a search of Applab.palette, but can't find anywhere it's used directly, although I know it is because it controls the JSON showed in levelbuilder. Going to leave as-is for now since I know it works.